src/python/cli_new/lib/cli/http.py

```diff
@@ -37,6 +37,7 @@ def read_endpoint(addr, endpoint, config, query=None):
 
     try:
         addr = cli.util.sanitize_address(addr)
+
```

Comment:
Nit: why the newline?

Reply:
No reason. :man_shrugging:  will remove it.